compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -433,7 +434,11 @@ std::string t_go_generator::camelcase(const std::string& value) const {
       if (islower(value2[i + 1])) {
         value2.replace(i, 2, 1, toupper(value2[i + 1]));
       }
-      fix_common_initialism(value2, i);
+
+	  if (i > std::numeric_limits<int>().max()) {
+        throw "integer overflow in t_go_generator::camelcase, value = " + value;
```

Comment:
I was using Visual Studio 2010 for many of these changes, which was inserting tabs.  I will clean this up.